nautobot/extras/management/commands/runjob.py

```diff
@@ -28,10 +32,17 @@ def add_arguments(self, parser):
             "--username",
             help="User account to impersonate as the requester of this job",
         )
+        parser.add_argument("-d", "--data", type=str, help="Provide json serializable data to be passed to job.")
+        parser.add_argument(
+            "-t",
+            "--testjob",
```

Comment:
I'd prefer something like `-l/--local` personally.

Reply:
+1 for `-l/--local`. Testing is one use case, but I can imagine running locally "for real" is also perfectly valid.

Reply:
@lampwins I was trying to be respectful of the utility importing for a testing folder (not tests though) and I wasn't sure in general if we could/should consider this a supported use case. Would it make sense to move this to a normal utility folder?

Reply:
@itdependsnetworks sorry I didn't look deeply at how this is actually implemented, but I do think the command has merit, so we should continue with this.

I think we should not use `run_job_for_testing` because it has some really unsafe logic for handing users in a non-testing scenario. Plus, with @gsnider2195's recent work with change contexts, there is a lot of boilerplate in there that can be refactored anyway. We probably want to write a better utility for this use case. Sorry, I know this expands your original intent, but I think it is the right thing to do.

Reply:
Yea, that is fair. Can you just be a bit more specfic about path forward? e.g. 

* Such as, add a function `/nautobot/utilities/<filename>.py` called `<name>`
* This should copy the functionality of `run_job_for_testing` but be aware of X

Reply:
I looked quickly, will def need advice on path forward. I am contemplating breaking up into 2 PRs, 1 for the data and template update, and then local for the second one. Will likely do that if I don't hear anything in the am.